source/renderer/app/components/widgets/FullyDecentralizedEffect.scss

```diff
@@ -0,0 +1,17 @@
+.component {
+  height: 100%;
+  left: 0;
+  position: absolute;
+  top: 0;
+  width: 100%;
+  z-index: 0;
+  //
+  // background: red;
```

Comment:
@daniloprates can you remove this please

Reply:
Why don't you like red bgs 😅  (done)

Reply:
hahahahah I'm doing that also 😄